Assets/Mirror/Examples/MultipleAdditiveScenes/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,119 @@
+    public class PlayerController : NetworkBehaviour
+    {
+        public CharacterController characterController;
+        public CapsuleCollider capsuleCollider;
+        public Rigidbody rigidbody3D;
+
+        void OnValidate()
+        {
+            if (characterController == null)
+                characterController = GetComponent<CharacterController>();
+            if (capsuleCollider == null)
+                capsuleCollider = GetComponent<CapsuleCollider>();
+            if (rigidbody3D == null)
+                rigidbody3D = GetComponent<Rigidbody>();
+        }
+
+        void Start()
+        {
+            capsuleCollider.enabled = isServer;
+        }
+
+        public override void OnStartLocalPlayer()
+        {
+            base.OnStartLocalPlayer();
+
+            Camera.main.orthographic = false;
+            Camera.main.transform.SetParent(transform);
+            Camera.main.transform.localPosition = new Vector3(0f, 3f, -8f);
+            Camera.main.transform.localEulerAngles = new Vector3(10f, 0f, 0f);
+        }
+
+        void OnDisable()
+        {
+            if (isLocalPlayer && Camera.main != null)
+            {
+                Camera.main.orthographic = true;
+                Camera.main.transform.SetParent(null);
+                SceneManager.MoveGameObjectToScene(Camera.main.gameObject, SceneManager.GetActiveScene());
+                Camera.main.transform.localPosition = new Vector3(0f, 70f, 0f);
+                Camera.main.transform.localEulerAngles = new Vector3(90f, 0f, 0f);
+            }
+        }
+
+        [Header("Movement Settings")]
+        public float moveSpeed = 8f;
+        public float turnSensitivity = 5f;
+        public float maxTurnSpeed = 150f;
+
+        [Header("Diagnostics")]
+        public float horizontal;
+        public float vertical;
+        public float turn;
+        public float jumpSpeed;
+        public float jumpTime;
+        public bool isGrounded = true;
+        public bool isFalling;
+        public Vector3 velocity;
+
+        void Update()
+        {
+            if (!isLocalPlayer)
+                return;
+
+            horizontal = Input.GetAxis("Horizontal");
+            vertical = Input.GetAxis("Vertical");
+
+            // Q and E cancel each other out, reducing the turn to zero
+            if (Input.GetKey(KeyCode.Q))
+                turn = Mathf.MoveTowards(turn, -maxTurnSpeed, turnSensitivity);
+            if (Input.GetKey(KeyCode.E))
+                turn = Mathf.MoveTowards(turn, maxTurnSpeed, turnSensitivity);
+            if (Input.GetKey(KeyCode.Q) && Input.GetKey(KeyCode.E))
+                turn = Mathf.MoveTowards(turn, 0, turnSensitivity);
+            if (!Input.GetKey(KeyCode.Q) && !Input.GetKey(KeyCode.E))
+                turn = Mathf.MoveTowards(turn, 0, turnSensitivity);
+
+            if (isGrounded)
+                isFalling = false;
+
+            if ((isGrounded || !isFalling) && jumpSpeed < 1f && Input.GetKey(KeyCode.Space))
```

Comment:
add {} please for consistency with the else case

Reply:
Done